packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -123,6 +123,10 @@ class ImageCache {
   int get currentSize => _cache.length;
 
   /// Maximum size of entries to store in the cache in bytes.
+  /// This cache stores decompressed images, so the size of each entry
+  /// in bytes is based on the number of pixels in the image, which is often
+  /// much higher than its compressed size. This value should therefore be
```

Comment:
nit: remove "This value should therefore be selected with that in mind." That is kinda obvious.